.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,3 +13,5 @@ Resolves #
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
 - [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
 - [ ] `make vendor` does not cause changes.
+- [ ] Godoc comments have been added to (1) new code additions, and (2) existing code which has been
```

Comment:
What about removing the `Godoc` here? So it is kept applicable for "all" languages used (e.g. Golang, Bash and others possibly).